h2o-algos/src/test/java/hex/knn/KNNDistanceTaskTest.java

```diff
@@ -0,0 +1,97 @@
+package hex.knn;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.util.TwoDimTable;
+
+public class KNNDistanceTaskTest extends TestUtil {
```

Comment:
I am missing asserts on the values you get from the methods. You just assert that something exists but you don't know if its correct. I would run it on some toy data also and check the values.

Reply:
I check that the values are correct in KNNTest, but it could also be here.